pages/index.html

```diff
@@ -5,10 +5,20 @@
 
 <article>
   <h2>Fedora 31 is now available!</h2>
-  <p class="author"><time datetime="2019-01-01">November 1, 2019</time></p>
+  <p class="author"><time datetime="2019-11-01">November 1, 2019</time></p>
 
   <p>Please <a href="https://silverblue.fedoraproject.org/download">download</a> Fedora Silverbue 31 and let us know what you think about this release!</p>
 
+    <img class="blog-image" alt="Silverblue logo" src="/public/silverblue-logo.svg">
+
+  <p>This release includes some exciting improvements, such as:
+    <ul>
+			<li>Several toolbox <a href="https://debarshiray.wordpress.com/2019/11/01/toolbox-a-fall-2019-update/">optimisations</a> and tighter integrations with libpod (podman) and the host system
+	  <li>GNOME being updated to version 3.34
+      <li>CgroupsV2 becoming our default control groups
```

Comment:
it might be worth us linking to the common bugs related to this change:

- https://fedoraproject.org/wiki/Common_F31_bugs#Podman_fails_to_run_containers_on_upgraded_systems_.28due_to_use_of_runc_runtime_with_cgroups_v2.29
- https://fedoraproject.org/wiki/Common_F31_bugs#Docker_package_no_longer_available_and_will_not_run_by_default_.28due_to_switch_to_cgroups_v2.29

Reply:
I have added known issues and improved a bit the text